shopfloor_reception/services/reception.py

```diff
@@ -768,25 +763,32 @@ def _align_product_uom_qties(self, move):
         # If move is already done, do not update lines qties
         # if move.state in ("done", "cancel"):
         #     return
-
+        move = line.move_id
         qty_todo = move.product_uom_qty
-        qty_done = sum(move.move_line_ids.mapped("qty_done"))
+        qty_done = 0.0
+        move_uom = move.product_uom
+        for move_line in move.move_line_ids:
+            # Use move's uom
+            qty_done += move_uom._compute_quantity(
+                move_line.qty_done, move_line.product_uom_id, round=False
+            )
```

Comment:
Should we not use the move_line_uom
```suggestion
            line_uom = move_line.product_uom_id
            qty_done += line_uom._compute_quantity(
                move_line.qty_done, move_uom.product_uom_id, round=False
            )
```

Reply:
Fixed